src/types/koa.d.ts

```diff
@@ -0,0 +1,7 @@
+import * as router from 'koa';
```

Comment:
nit: I wonder if this file shuold rather be the name of the module, such us `koa-json-body.d.ts`?

Reply:
ok, thanks for the explanation! My rationale was that this change is to accomodate the code from the module `koa-json-body`.  I'm not sure what the best practices in the TS world are in this regard, but anyway this sounds good to me :) Nothing set in stone anyway.